src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -466,15 +428,35 @@ public static boolean isCurrentUserCPMAdmin(String userId, Environment env) {
 	}
 
 	/*
-	 * get CoSign user
+	 * get CoSign user, for local development user will be passed from url parameter as
+	 * http://localhost:8080/?testUser=<uniqname> together with use.testUser.url from the
+	 * application.properties we will determine valid user. In short this block is mimicking
+	 * cosign user if CoSign is not enabled
 	 */
 	public static String getRemoteUser(HttpServletRequest request, Environment env) {
-		String propertyRemoteUser = env.getProperty(TEST_REMOTEUSER);
+		String user = null;
```

Comment:
This `testUser` parameter is mandatory for both admin and non-admin user. `testUser` is mimicking cosignUser when cosign disabled.